### ISSUE TITLE
Unhelpful Error on enc 

### ISSUE THREAD

**Issue by brizzbuzz** [Just-Insane/helm-vault#24]
## Describe the bug

I am trying to follow along with a tutorial and test out `helm-vault` but am experiencing an error that is _pretty_ tough to grok.  I have a local Vault running, have my env vars set, but when I run 

`helm vault enc webgo/values.yaml`

I get the following error

```
❯ helm vault enc webgo/values.yaml
Input a value for /image/repository: test
ERROR: /Users/lappy/University/helm-starter-go/charts
```

## To Reproduce

I'm _really_ not sure.  I don't know if this is a local problem.  I don't know if it's a syntax problem (though I doubt this because I've deployed the chart with no secrets).  I just don't know where to start ☹️

## Expected behavior

Either a success or at least a better description of what is wrong

## Environment:

 * OS: Mac Big Sur
 * Python Version 3.9
 * Vault Version 1.6.3

**Comment by brizzbuzz** [comment]
Aha yep that was it, I'm getting a different error now 

```
Error: {"request_id":"6a08df69-3f01-0bcd-a885-0b798bc2923b","lease_id":"","renewable":false,"lease_duration":0,"data":null,"wrap_info":null,"warnings":["Invalid path for a versioned K/V secrets engine. See the API docs for the appropriate API endpoints to use. If using the Vault CLI, use 'vault kv put' for this operation."],"auth":null}
```

but this at least I can work with 😄 assuming this has to do with my vault instance configuration.  

Cheers for the quick response, going to leave this open for now, keying off the git path is not _inherently_ a bad idea because most of the time this should be run from a git repo.  though it will trip up newbies like me are tinkering around in a test directory that isn't versioned

**Comment by Just-Insane** [comment]
That error looks like a mismatch between version 1 and version 2 of the K/V secrets engine, which is an easy enough fix. The plugin defaults to V1, but Vault defaults to V2.

I'm not too sure what a good solution would be if I remove the git check to be honest. I think the best bet might be to just update the documentation and error message to be more descriptive.

**Comment by brizzbuzz** [comment]
Yea the git path makes sense because (assuming it's relative?) it should be portable across systems.  Maybe the best bet is to have it be configurable? Git by default, and print an error message if no git path is found, saying that you need to manually enter a `--path` flag or something if it's not in a git repo

**Comment by Just-Insane** [comment]
Fixed in release v0.2.3.
